src/openforms/submissions/tests/test_tasks_confirmation_emails.py

```diff
@@ -573,19 +578,88 @@ def test_template_is_rendered_in_submission_language(self):
                         "confirmationRecipient": True,
                     },
                 ],
-                submitted_data={"email": "abuse@example.com"},
+                submitted_data={
+                    "email": "abuse@example.com",
+                    "fubar": "Haveibeenpwned.com123!@#",
+                    "editgrid1": [
+                        {
+                            "radio1": "radiov2",
+                        },
+                        {
+                            "radio1": "radiov1",
+                            "select1": "selectv2",
+                        },
+                    ],
+                },
                 form__generate_minimal_setup=True,
                 form__translation_enabled=True,
                 form__name="Translated form name",
                 form__formstep__form_definition__name="A Quickstep",
+                form__formstep__form_definition__component_translations={
+                    "en": {
+                        "Untranslated Repeating Group label": "Translated Repeating Group label",
+                        "Untranslated Repeating Group Item label": "Translated Repeating Group Item label",
+                        "Untranslated Radio option 1": "Translated Radio option 1",
+                        "Untranslated Radio option 2": "Translated Radio option 2",
+                        "Untranslated Select option 2": "Translated Select option 2",
+                    },
+                },
                 form__formstep__form_definition__configuration={
                     "components": [
                         {
                             "key": "foo",
                             "type": "textfield",
-                            "label": "Foo",
+                            "label": "Label with no translation",
+                            "showInEmail": True,
+                        },
+                        {
+                            "key": "fubar",
+                            "type": "password",
+                            "label": "Label with no translation",
+                            # TODO add a constraint, I don't want this state in my db!
```

Comment:
not sure I understand, but since this stored in a JSONField and the datastructure itself is recursive/arbitrary nested, I don't think it's possible to express a database constraint for this at all.